website/static/js/commentpane.js

```diff
@@ -11,7 +11,10 @@
         onOpen: function() {}
     };
 
-    var CommentPane = CommentPane || function(selector, options) {
+    /*
+       mode: 'widget', 'pane' or 'page'
+     */
+    var CommentPane = CommentPane || function(selector, hostPage, hostName, mode, options) {
```

Comment:
I'm not saying to remove them completely; I mean remove them from this module; they should be managed by the comment controller/viewmodel, not the comment pane.